src/plugins/integrations/AWSIntegration.ts

```diff
@@ -200,7 +204,6 @@ class AWSIntegration implements Integration {
                 [SpanTags.OPERATION_TYPE]: operationType ? operationType : 'READ',
                 [AwsS3Tags.BUCKET_NAME]: request.params.Bucket,
```

Comment:
What if `request.params.Bucket` is undefined? I mean, there is no `Bucket` in the request

Reply:
Then `AwsS3Tags.BUCKET_NAME` will be undefined and it will not be serialized during reporting.